サインアウト/サインイン機能の追加

drumnaoto opened the pull request:
# WHAT
サインアウト/サインイン機能の追加。

# WHY
必須機能の為。

drumnaoto committed:
add-sign-in-sign-out

サインインしていない時はログイン画面にリダイレクトするようにしました。

Atsuyoshi-N reviewed (commented):
user_edit のパスに飛ばせるように変更お願いいたします。
usernameの表示ができるように変更をお願いいたします。

drumnaoto committed:
chage-index-view

indexビューにて、
ログインしているユーザーのnameが表示され、
歯車アイコンを押したら、
ログインしているユーザーのedit画面に行くように編集しました。

drumnaoto committed:
change-a-to-i

Font Awesomeの表示の仕方をaからiに変更したのに伴い、
scssの方も変更しました。

Atsuyoshi-N commented on app/views/messages/index.html.haml:
```diff
@@ -2,12 +2,14 @@
       %ul.header__lists
         %li
-          %a.fa.fa-cog{ href:"https://tech-camp.in/expert"}
+          =link_to edit_user_path(current_user.id) do
```
current_userだけでも取れると思いますので、修正お願いいたします。

drumnaoto committed:
only-current_user

.idを消しました

Atsuyoshi-N reviewed (commented):
LGTMです！
